zeppelin-web/package.json

```diff
@@ -40,50 +38,53 @@
     "angular-sanitize": "1.5.7",
     "angular-touch": "1.5.7",
     "angular-ui-ace": "0.2.3",
+    "angular-ui-bootstrap": "~2.5.0",
+    "angular-ui-grid": "4.4.6",
     "angular-websocket": "1.1.0",
     "angular-xeditable": "0.1.12",
+    "ansi_up": "^2.0.2",
+    "ap-viewport-watch": "0.1.43",
     "bootstrap": "~3.4.1",
     "bootstrap3-dialog": "~1.35.4",
     "clipboard": "~1.5.5",
     "d3": "^3.4.4",
+    "diff": "3.3.0",
+    "diff-match-patch": "1.0.0",
     "es5-shim": "~3.1.0",
     "excel-builder": "2.0.0",
+    "github-markdown-css": "2.6.0",
+    "grunt-angular-templates": "^0.5.7",
+    "grunt-dom-munger": "^3.4.0",
+    "headroom.js": "^0.9.3",
     "highlight.js": "9.4.0",
     "jquery": ">=1.8",
     "jquery-ui": ">=1.8",
     "jquery.scrollto": "~2.1.2",
-    "diff": "3.3.0",
     "json3": "~3.3.1",
     "jszip": "2.6.1",
     "lodash": "~3.9.3",
+    "mathjax": "^2.7.9",
```

Comment:
It looks like the dependency version has changed. It seems to be related to #4786, so wouldn’t it be better to keep the existing version in this PR?